docs/source/user_guide/config_file.rst

```diff
@@ -24,69 +24,143 @@ The config file contains several sections:
+* ``training.min_epochs`` / ``training.max_epochs`` (*int, default: 300*): length of training.
   An epoch is one full pass through the dataset.
   As an example, if you have 400 labeled frames, and ``training.train_batch_size=10``, then your
   dataset is divided into 400/10 = 40 batches.
   One "batch" in this case is equivalent to one "iteration" for DeepLabCut.
   Therefore, 300 epochs, at 40 batches per epoch, is equal to 300*40=12k total batches
   (or iterations).
 
-.. _config_num_gpus:
-* ``training.num_gpus``: the number of GPUs for :ref:``multi-GPU training <multi_gpu_training>``.
+* ``training.log_every_n_steps`` (*int, default: 10*): frequency to log training metrics for
+  tensorboard (one step is one batch)
+
+* ``training.check_val_every_n_epochs`` (*int, default: 5*): frequency to log validation metrics
+  for tensorboard
+
+* ``training.ckpt_every_n_epochs`` (*int or null, default: null*): save model weights every n
+  epochs; must be divisible by ``training.check_val_every_n_epochs`` above.
+  If null, only the best weights will be saved after training, where "best" is defined as the
+  weights from the epoch with the lowest validation loss.
+
+* ``training.early_stopping`` (*bool, default: false*): if false, the default is to train for the
+  max number of epochs and save out the best model according to the validation loss; if true, early
+  stopping will exit training if the validation loss continues to increase for a given number of
+  validation checks (see ``training.early_stop_patience`` below).
+
+* ``training.early_stop_patience`` (*int, default: 3*): number of validation checks over which to
+  assess validation metrics for early stopping; this number, multiplied by
+  ``training.ckpt_every_n_epochs``, gives the number of epochs over which the validation loss must
+  increase before exiting.
+
+* ``training.rng_seed_data_pt`` (*int, default: 0*): rng seed for splitting labeled data into
+  train/val/test
 
-* ``training.accumulate_grad_batches``: (experimental) number of batches to accumulate gradients
-  for before updating weights. Simulates larger batch sizes with memory-constrained GPUs. This
-  parameter is not included in the config by default and should be added manually to the
+* ``training.rng_seed_model_pt`` (*int, default: 0*): rng seed for weight initialization of the head
+
+* ``training.lr_scheduler`` (*str, default: multisteplr*): reduce the learning rate by a certain
+  factor after a given number of epochs (see ``training.lr_scheduler_params.multisteplr`` below)
+
+* ``training.lr_scheduler_params.multistep_lr``: milestones: epochs at which to reduce learning
+  rate; gamma: factor by which to multiply learning rate at each milestone
+
+* ``training.uniform_heatmaps_for_nan_keypoints`` (*bool, default: true*): how to treat missing
+  hand labels; false to drop, true to force uniform heatmaps. True will lead to better confidence
+  values, while false allows for incompletely labeled data.
```

Comment:
I'm still unclear on why you'd want to use false

Reply:
yes this is a great idea - mind writing this down in the v2.0 notes?

Reply:
also I tried to clarify the nan handling explanation a bit, and when you might want to use false

Reply:
Added!